x-pack/plugins/reporting/server/core.ts

```diff
@@ -357,30 +362,44 @@ export class ReportingCore {
     return startDeps.esClient;
   }
 
-  public getScreenshots(options: ScreenshotOptions): Rx.Observable<ScreenshotResult> {
+  private mapToScreenshotConfigOptions(options: PdfScreenshotOptions | PngScreenshotOptions) {
```

Comment:
I suppose we can revert changes in this file since there are no longer two separate methods. Or am I missing something?

Reply:
Because we are wrapping a function with a generic options object we need to map our options in reporting to the options screenshotting is using.

I've collapsed this code a bit by using method polymorphism.